### DISCUSSION TITLE
Feature Request: runCmd API function

### DISCUSSION THREAD

**Discussion by zorn-v** [ueberdosis/tiptap#5201] [Feature Requests]
### Description

Listing docs and little frustrated about `editor.chain().focus().setFontFamily('Inter').run()` for every command.
I definetly create function like
```js
function runCmd(cb) {
  return cb(editor.chain().focus()).run()
}
// use like this
runCmd((cmd) => cmd.setFontFamily('Inter'))
```
in my project

So why not offer something like `editor.runCmd((cmd) => cmd.setFontFamily('Inter'))`

### Use Case

Just to reduce boilerplate

### Type

New Tiptap API

**Comment by nperez0111** [comment]
We already have the `editor.commands` api that does exactly as you've describe: https://tiptap.dev/docs/editor/api/commands

**Reply by nperez0111** [reply]
Chaining, is powerful in that it lets you do multiple commands in a single prosemirror transaction. This is helpful for a number of reasons, one small one is if something in the chain can not be applied they are all not applied. 

So you'll see the chain API used often because of it's flexibility. Not boilerplate, but neccessary complexity

**Reply by zorn-v** [reply]
> We already have the editor.commands api that does exactly as you've describe

Most commands need focus, so chain should be used.

**Reply by nperez0111** [reply]
*most*, not all. Feel free to use your own helpers, tiptap is meant to be a flexible framework.
